### DISCUSSION TITLE
student developer pack approved but can't access my copilot

### DISCUSSION THREAD

**Discussion by bharat2004** [community/community#146050] [GitHub Education]
### Hi everyone,

My student developer pack request was approved but i can't get access my github copilot and in the plans section of my github it still see free plan but why and my approval was made about 2 days ago
![Screenshot 2024-12-02 200459](https://github.com/user-attachments/assets/eb202e81-1153-4ea9-8ba3-ec7189f31a5f)
See this screenshot

My github id is bharat2004

Kindly help me
Thank you

**Comment by davevad93** [comment]
Hi @bharat2004 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @bharat2004 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
